.github/workflows/docker-image.yml

```diff
@@ -6,6 +6,10 @@ on:
       - main
       - dev
       - staging
+  pull_request:
+    branches:
+      - main
+      - dev
```

Comment:
i know we don't have a "staging" right now, but can we add it here just for consistency in case that changes.